5.machine_learning/machine_learning.sh

```diff
@@ -1,23 +1,17 @@
 #!/bin/bash
 
-# initialize the correct shell for your machine to allow conda to work (see README for note on shell names)
-conda init bash
-# activate the main conda environment
-conda activate cfret_data_env
+# Get the directory where this script is located
```

Comment:
This bash script would benefit from a docstring - what is the purpose?

Reply:
Good point, my goal for this was to have one script that would run the other scripts. Let me add that, thanks!

Reply:
I am running into some issues with that script so I have removed it for now, I'll try fixing it later, having some pathing issues with conda.